docs/sources/flow/reference/components/loki.echo.md

```diff
@@ -41,6 +41,38 @@ Name | Type | Description
 
 `loki.echo` does not expose any component-specific debug information.
 
+<!-- START GENERATED COMPATIBLE COMPONENTS -->
```

Comment:
Here just for fun I put the section in a different place rather than at the end. The auto-updating will add the section at the end if it doesn't exist, but it can be moved around and tests will still detect it and update it in its existing location.